src/Features/Core/Portable/UnifiedSuggestions/UnifiedPredefinedSuggestedActionCategoryNames.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+namespace Microsoft.CodeAnalysis.UnifiedSuggestions
+{
+    /// <summary>
+    /// Equivalent to PredefinedSuggestedActionCategoryNames, but in a location that
```

Comment:
How will we ensure these two types always stay in sync?

Reply:
Discussed with Manish - long term it would probably be good to have a shared interface. But I'm not sure that's possible in this PR.

Reply:
i don't think it actually matters.  because we don't pass these values along untouched.  instead, they always go through a conversion helper to ensure they match the final host values.